.circleci/config.yml

```diff
@@ -12,9 +12,10 @@ jobs:
     environment:
       POSTGRES_TEST_HOST: localhost
       POSTGRES_TEST_USER: root
-      POSTGRES_TEST_PASS: ''
+      DBT_ENV_SECRET_POSTGRES_TEST_PASS: ''
```

Comment:
> Redshift, snowflake and bigquery tests currently fail because I changed the env vars.  Circle CI needs the following vars added:
> - `DBT_ENV_SECRET_REDSHIFT_TEST_PASS` (was `REDSHIFT_TEST_PASS`)
> - `DBT_ENV_SECRET_SNOWFLAKE_TEST_PASS` (was `SNOWFLAKE_TEST_PASSWORD`)
> - `DBT_ENV_SECRET_BIGQUERY_SERVICE_KEY_PATH` (was `BIGQUERY_SERVICE_KEY_PATH`)

@emmyoop In order to unlock this PR with the least amount of additional effort, could we change the names of the environment variables in a separate PR?

Reply:
We can!  I'm currently using it for some testing and need the env vars named as is while I iterate with it.

Reply:
I realized I could leave the CI vars as is since I moved the profiles for the purposes of my testing.